.github/workflows/publish.yml

```diff
@@ -28,13 +28,11 @@ jobs:
 
       - name: Install go
         run: |
-          curl -L -o /tmp/go.pkg https://golang.org/dl/go1.16.darwin-amd64.pkg
+          curl -L -o /tmp/go.pkg https://golang.org/dl/go1.17.darwin-amd64.pkg
```

Comment:
The [macOS](https://github.com/actions/virtual-environments/blob/main/images/macos/macos-10.15-Readme.md) and [Ubuntu](https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md) runners have Go 1.17.3 cached. If none of the dependencies require building on Darwin, you could switch over to Ubuntu runners at 10% of the price.

Reply:
Hey @williamtheaker - I didn't set this up, but I think the `build.sh` [step depends on a tool called `lipo` to make a universal macOS binary](https://github.com/Gusto/committer/blob/master/.github/build.sh#L12) (arm64 + amd64). This tool is from Xcode, so I imagine it's macOS only.